projects/js-packages/publicize-components/src/components/share-status/modal-trigger.tsx

```diff
@@ -16,8 +17,21 @@ type ModalTriggerProps = ButtonProps & {
 export const ModalTrigger = forwardRef(
 	( { withWrapper = false, ...props }: ModalTriggerProps, ref: unknown ) => {
 		const { openShareStatusModal } = useDispatch( socialStore );
-
 		const featureFlags = useSelect( select => select( socialStore ).featureFlags(), [] );
+		const { shareStatus } = useSelect( select => {
+			const store = select( socialStore );
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any -- `@wordpress/editor` is a nightmare to work with TypeScript
+			const _editorStore = select( editorStore ) as any;
+
+			return {
+				shareStatus: store.getPostShareStatus( _editorStore.getCurrentPostId() ),
```

Comment:
You don't need to pass the post ID because it defaults to the current post after #39112

Reply:
Ah, thanks, didnt see that change

Reply:
So it can be a oneliner

```ts
const shareStatus = useSelect( select => select( socialStore ).getPostShareStatus(), [] );
```

Reply:
Should we merge the 2 useSelects, as both are from the social store? Or keep them as 2 one-liners? I mean the feature flag one above

Reply:
Oneliners are pretty